src/js/utils/MesosStateUtil.js

```diff
@@ -2,8 +2,34 @@ import Util from './Util';
 
 const RESOURCE_KEYS = ['cpus', 'disk', 'mem'];
 
+function setSchedulerName(name, tasks) {
+  return tasks.map(function (task) {
+    let newTask = Object.assign({}, task);
+
+    newTask.scheduler = name;
+
+    return newTask;
+  });
+}
+
 const MesosStateUtil = {
 
+  assignSchedulerNameToTasks(state) {
```

Comment:
@mlunoe switched back to `isStartedByMarathon` approach. Let's discuss the PR today quickly in Slack, so we get it through.